### PR TITLE
Clean up IconSelectorView code

### PR DESCRIPTION
**Pull request by EvilOne**
We all know you love app icons! 😃 Let's make it a bit easier to add them going forward and stop adding a new `Section` to call out the great contributors.

### CODE REVIEW

File: IceCubesApp/App/Tabs/Settings/IconSelectorView.swift [hunk 1]
```diff
@@ -60,6 +38,22 @@ struct IconSelectorView: View {
     }
   }
 
+  struct IconSelector: Identifiable {
```
Comment (EvilOne) [inline IceCubesApp/App/Tabs/Settings/IconSelectorView.swift:41]: I couldn't come up with a good name.

File: IceCubesApp/App/Tabs/Settings/IconSelectorView.swift [hunk 2]
```diff
@@ -60,6 +38,22 @@ struct IconSelectorView: View {
...
+    let icons: [Icon]
+
+    static let items = [
+      IconSelector(title: "Official icons", icons: [.primary, .alt1, .alt2, .alt3, .alt4, .alt5, .alt6, .alt7, .alt8,
+                                                     .alt9, .alt10, .alt11, .alt12, .alt13, .alt14,
+                                                     .alt15, .alt16, .alt17, .alt18, .alt19, .alt25]),
+      IconSelector(title: "Icons by Albert Kinng", icons: [.alt20, .alt21, .alt22, .alt23, .alt24]),
+      IconSelector(title: "Icons by Dan van Moll", icons: [.alt26, .alt27, .alt28]),
+      IconSelector(title: "Icons by @te6-in (GitHub)", icons: [.alt29, .alt30, .alt31, .alt32]),
+      IconSelector(title: "Icon by W. Kovács Ágnes (@wildgica)", icons: [.alt23]),
```
Comment (icod) [inline IceCubesApp/App/Tabs/Settings/IconSelectorView.swift:53]: Looking at the old code, I believe that 23 is one of Alberts. Ágnes did _33_. 

```suggestion
      IconSelector(title: "Icon by W. Kovács Ágnes (@wildgica)", icons: [.alt33]),
```

Comment (EvilOne) [reply IceCubesApp/App/Tabs/Settings/IconSelectorView.swift]: Thanks!

**Comment by Dimillian** [comment]
hahaha, thanks!

**Pull request merged**